.phpunit.result.cache

```diff
@@ -0,0 +1 @@
angeTest::testRangePass";d:0;s:18:"DateTest::testFail";d:0;s:18:"DateTest::testPass";d:0;s:22:"DateTimeTest::testFail";d:0;s:22:"DateTimeTest::testPass";d:0;s:40:"DateWithinRangeTest::testWithinRangeFail";d:0;s:40:"DateWithinRangeTest::testWithinRangePass";d:0;s:17:"EinTest::testFail";d:0;s:17:"EinTest::testPass";d:0;s:19:"EmailTest::testFail";d:0.001;s:19:"EmailTest::testPass";d:0;s:19:"FloatTest::testFail";d:0;s:19:"FloatTest::testPass";d:0;s:25:"FloatTest::testStringPass";d:0;s:23:"IPAddressTest::testFail";d:0;s:23:"IPAddressTest::testPass";d:0;s:21:"IntegerTest::testFail";d:0;s:21:"IntegerTest::testPass";d:0;s:18:"NameTest::testFail";d:0;s:18:"NameTest::testPass";d:0;s:26:"NotEmptyTest::testNotEmpty";d:0;s:24:"NotNullTest::testNotNull";d:0;s:36:"NumberWithinRangeTest::testRangePass";d:0;s:36:"NumberWithinRangeTest::testRangeFail";d:0;s:31:"NumericRangeTest::testRangePass";d:0;s:31:"NumericRangeTest::testRangeFail";d:0;s:21:"NumericTest::testPass";d:0.006;s:21:"NumericTest::testFail";d:0;s:25:"PhoneNumberTest::testFail";d:0;s:25:"PhoneNumberTest::testPass";d:0;s:29:"PhoneNumberTest::testIntlFail";d:0;s:29:"PhoneNumberTest::testIntlPass";d:0;s:25:"PolicyTest::testDateRange";d:0.016;s:24:"PolicyTest::testCurrency";d:0;s:22:"PositiveTest::testFail";d:0;s:22:"PositiveTest::testPass";d:0;s:26:"StringLengthTest::testFail";d:0;s:26:"StringLengthTest::testPass";d:0;s:20:"StringTest::testFail";d:0;s:20:"StringTest::testPass";d:0;s:18:"TimeTest::testFail";d:0;s:18:"TimeTest::testPass";d:0;s:19:"UrlTest1::testFail1";d:0;s:18:"UrlTest1::testPass";d:0;s:26:"TemporaryTest::testGetFile";d:0;s:29:"WebHookPostTest::testOpenFail";d:0;s:29:"WebHookPostTest::testOpenPass";d:0;s:25:"CSVFormatTest::testFormat";d:0;s:20:"HTMLTest::testFormat";d:0;s:20:"JsonTest::testFormat";d:0;s:25:"PlainTextText::testFormat";d:0;s:20:"LogMuxTest::testPass";d:0;s:20:"LogMuxTest::testFail";d:0;s:26:"LogSingletonTest::testPass";d:0;s:26:"LogSingletonTest::testFail";d:0;s:27:"LogSingletonTest::testDebug";d:0;s:26:"LogSingletonTest::testInfo";d:0;s:29:"LogSingletonTest::testWarning";d:0;s:27:"LogSingletonTest::testError";d:0;s:27:"LogSingletonTest::testFatal";d:0;s:20:"LoggerTest::testPass";d:0;s:20:"LoggerTest::testFail";d:0;s:16:"CSVTest::testCSV";d:0.001;s:25:"LastFirstMITest::testPass";d:0;s:24:"PositionalTest::testPass";d:0;s:29:"CriteriaTest::testAndCriteria";d:0.001;s:28:"CriteriaTest::testOrCriteria";d:0;s:29:"CriteriaTest::testNotCriteria";d:0;s:23:"RegistrylTest::testPass";d:0;s:23:"RegistrylTest::testFail";d:0;s:20:"MemoryTest::testPass";d:0;s:23:"DatelTest::testDateOnly";d:0;s:25:"DatelTest::testDaysAsText";d:0;s:35:"DatelTest::testDifferenceUnitAsText";d:0;s:23:"DatelTest::testLeapYear";d:0;s:26:"DatelTest::testDaysInMonth";d:0;s:19:"DatelTest::testDiff";d:0;s:27:"DatelTest::testSubtractDays";d:0;s:22:"LanguageTest::testLoad";d:0;s:20:"TimerTest::testTimer";d:0;s:20:"TimerTest::testReset";d:2;s:23:"WebHookTest::testGet200";d:2.216;s:23:"WebHookTest::testGet401";d:0.408;}}}
```

Comment:
Should this file be part of the repository?

Reply:
Not sure, it was auto-generated. Will remove.